tooling-tests/src/test/resources/test-src/generated-sources/tool_test_server_3/helloworld_service.bal

```diff
@@ -2,7 +2,7 @@ import ballerina/grpc;
 
 listener grpc:Listener ep = new (9090);
 
-@grpc:ServiceDescriptor {descriptor: ROOT_DESCRIPTOR_HELLOWORLDFLOAT, descMap: getDescriptorMapHelloWorldFloat()}
+@grpc:ServiceDescriptor {descriptor: HELLOWORLDFLOAT_DESC}
```

Comment:
Along with this effort, I am thinking of changing the service annotation to 
```
@grpc:Descriptor {value: HELLOWORLDFLOAT_DESC}
```
align with the new annotation without breaking the existing code. 
@BuddhiWathsala @dilanSachi @MadhukaHarith92 WDYT?

Reply:
+1, looks more consistent.

Reply:
+1

Reply:
At runtime, we need to check for any of these two annotations and proceed to avoid breaking user code. 
The only scenario it breaks is when the user generates the stub for the existing services, which breaks with this change.

Reply:
Let's create another issue and handle it separately.